learn/tools-ides.md

```diff
@@ -1,53 +0,0 @@
----
```

Comment:
what is the impact of removing this page for the relevant plugin pages such as [1] , [2] as they are sub-paths of this page?

[1] https://ballerina.io/learn/tools-ides/intellij-plugin/
[2] https://ballerina.io/learn/tools-ides//intellij-plugin/

Reply:
Updated the "perma" and "redirect" links in the below pages to rectify the impact of the URLs paths:

- learn/how-to-document-ballerina-code.md
- learn/tools-ides/intellij-plugin.md
- learn/tools-ides/intellij-plugin/using-intellij-plugin-features.md
- learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
- learn/tools-ides/vscode-plugin.md
- learn/tools-ides/vscode-plugin/documentation-viewer.md
- learn/tools-ides/vscode-plugin/graphical-editor.md
- learn/tools-ides/vscode-plugin/language-intelligence.md
- learn/tools-ides/vscode-plugin/run-all-tests.md
- learn/tools-ides/vscode-plugin/run-and-debug.md
- v1-1/learn/tools-ides/intellij-plugin.md
- v1-1/learn/tools-ides/intellij-plugin/using-intellij-plugin-features.md
- v1-1/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
- v1-1/learn/tools-ides/vscode-plugin.md
-  v1-1/learn/tools-ides/vscode-plugin/documentation-viewer.md
- v1-1/learn/tools-ides/vscode-plugin/graphical-editor.md
- v1-1/learn/tools-ides/vscode-plugin/language-intelligence.md
- v1-1/learn/tools-ides/vscode-plugin/run-all-tests.md
- v1-1/learn/tools-ides/vscode-plugin/run-and-debug.md

Reply:
we can map `learn/tools-ides/intellij-plugin/` to `learn/intellij-plugin/` using permalink and redirect all calls from `learn/tools-ides/intellij-plugin/` to goto `learn/intellij-plugin/`... this can be done to other pages as well.

Reply:
Did the redirections for the below pages as well.

- v0-991/learn.md
- v0-991/learn/tools-ides.md
- v0-991/learn/tools-ides/intellij-plugin.md
- v0-991/learn/tools-ides/intellij-plugin/testing-and-debugging-the-plugin.md
- v0-991/learn/tools-ides/intellij-plugin/using-intellij-plugin-features.md
- v0-991/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
- v0-991/learn/tools-ides/vscode-plugin.md
- v0-991/learn/tools-ides/vscode-plugin/documentation-viewer.md
- v0-991/learn/tools-ides/vscode-plugin/graphical-editor.md
- v0-991/learn/tools-ides/vscode-plugin/language-intelligence.md
- v0-991/learn/tools-ides/vscode-plugin/run-all-tests.md
- v0-991/learn/tools-ides/vscode-plugin/run-and-debug.md
- v1-0/learn.md
- v1-0/learn/tools-ides.md
- v1-0/learn/tools-ides/intellij-plugin.md
- v1-0/learn/tools-ides/intellij-plugin/using-intellij-plugin-features.md
- v1-0/learn/tools-ides/intellij-plugin/using-the-intellij-plugin.md
- v1-0/learn/tools-ides/vscode-plugin.md
- v1-0/learn/tools-ides/vscode-plugin/documentation-viewer.md
- v1-0/learn/tools-ides/vscode-plugin/graphical-editor.md
- v1-0/learn/tools-ides/vscode-plugin/language-intelligence.md
- v1-0/learn/tools-ides/vscode-plugin/run-all-tests.md
- v1-0/learn/tools-ides/vscode-plugin/run-and-debug.md
- v1-1/learn.md